content/en/docs/usage/kubectl-plugin.md

```diff
@@ -13,24 +13,36 @@ While the kubectl plugin is supported, it is recommended to use
 [cmctl](../cmctl/) as this enables a better experience via tab auto-completion.
 
 ## Installation
+
 You need the `kubectl-cert-manager.tar.gz` file for the platform you're using, these can be found on our [GitHub releases page](https://github.com/jetstack/cert-manager/releases).
 In order to use the kubectl plugin you need its binary to be accessible under the name `kubectl-cert_manager` in your `$PATH`.
 
-### macOS/Linux
+### Linux/MacOS
+
 Run the following commands to set up the plugin:
+
 ```console
-$ curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/kubectl-cert_manager-linux-amd64.tar.gz
-$ tar xzf kubectl-cert-manager.tar.gz
-$ sudo mv kubectl-cert_manager /usr/local/bin
+OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/cmctl-$OS-$ARCH.tar.gz
+tar xzf kubectl-cert-manager.tar.gz
+sudo mv kubectl-cert_manager /usr/local/bin
 ```
 
+Supported OS and CPU architectures:
+
+- linux/amd64
+- linux/arm64
+- darwin/amd64
+- darwin/arm64
```

Comment:
```suggestion
- `linux/amd64`
- `linux/arm64`
- `darwin/amd64`
- `darwin/arm64`
```
This should make the spellchecker happy BUT since we also have `windows/amd64` binaries I think this list might cause confusion. I suggest you drop the list.


Thanks and sorry for the confusing output in CI. See:
 * #802

Reply:
OK done, thanks